# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lualdap/config-no-compat-5.1.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ruN lualdap-1.0-vanilla/Makefile lualdap-1.0/Makefile
--- lualdap-1.0-vanilla/Makefile	2005-06-10 15:23:14.000000000 +0200
+++ lualdap-1.0/Makefile	2006-03-14 19:01:03.000000000 +0100
@@ -6,14 +6,11 @@
 
 include $(CONFIG)
 
-OBJS= src/lualdap.o $(COMPAT_DIR)/compat-5.1.o
+OBJS= src/lualdap.o
 
 
 src/$(LIBNAME): $(OBJS)
-	export MACOSX_DEPLOYMENT_TARGET="10.3"; $(CC) $(CFLAGS) $(LIB_OPTION) -o src/$(LIBNAME) $(OBJS) -lldap
-
-$(COMPAT_DIR)/compat-5.1.o: $(COMPAT_DIR)/compat-5.1.c
-	$(CC) -c $(CFLAGS) -o $@ $(COMPAT_DIR)/compat-5.1.c
+	$(CC) $(CFLAGS) $(LIB_OPTION) -o src/$(LIBNAME) $(OBJS)
 
 install: src/$(LIBNAME)
 	mkdir -p $(LUA_LIBDIR)
diff -ruN lualdap-1.0-vanilla/config lualdap-1.0/config
--- lualdap-1.0-vanilla/config	2005-06-10 15:23:14.000000000 +0200
+++ lualdap-1.0/config	2006-03-14 18:45:46.000000000 +0100
@@ -1,22 +1,22 @@
 # Installation directories
 # System's libraries directory (where binary libraries are installed)
-LUA_LIBDIR= /usr/local/lib/lua/5.0
+# LUA_LIBDIR= /usr/local/lib/lua/5.0
 # Lua includes directory
-LUA_INC= /usr/local/include/lua5
+# LUA_INC= /usr/local/include/lua5
 # OpenLDAP includes directory
-OPENLDAP_INC= /usr/local/include
+# OPENLDAP_INC= /usr/local/include
 
 # OS dependent
-LIB_OPTION= -shared #for Linux
-#LIB_OPTION= -bundle -undefined dynamic_lookup #for MacOS X
+# LIB_OPTION= -shared #for Linux
+# LIB_OPTION= -bundle -undefined dynamic_lookup #for MacOS X
 
 LIBNAME= lib$T-$V.so
-COMPAT_DIR= ../compat/src
+# COMPAT_DIR= ../compat/src
 
 # Compilation parameters
-WARN= -O2 -Wall -fPIC -W -Waggregate-return -Wcast-align -Wmissing-prototypes -Wnested-externs -Wshadow -Wwrite-strings -ansi
-INCS= -I$(LUA_INC) -I$(OPENLDAP_INC) -I$(COMPAT_DIR)
-CFLAGS= $(WARN) $(INCS)
-CC= gcc
+# WARN= -O2 -Wall -fPIC -W -Waggregate-return -Wcast-align -Wmissing-prototypes -Wnested-externs -Wshadow -Wwrite-strings -ansi
+# INCS= -I$(LUA_INC) -I$(OPENLDAP_INC) -I$(COMPAT_DIR)
+# CFLAGS= $(WARN) $(INCS)
+# CC= gcc
 
 # $Id: config,v 1.2 2005/06/10 13:23:14 tomas Exp $
diff -ruN lualdap-1.0-vanilla/src/lualdap.c lualdap-1.0/src/lualdap.c
--- lualdap-1.0-vanilla/src/lualdap.c	2005-05-23 16:35:11.000000000 +0200
+++ lualdap-1.0/src/lualdap.c	2006-03-14 18:46:29.000000000 +0100
@@ -16,7 +16,6 @@
 
 #include "lua.h"
 #include "lauxlib.h"
-#include "compat-5.1.h"
 
 
 #define LUALDAP_PREFIX "LuaLDAP: "
